packages/@jsii/dotnet-runtime/src/Amazon.JSII.Runtime/Services/Runtime.cs

```diff
@@ -10,19 +9,14 @@ internal sealed class Runtime : IRuntime
         public Runtime(INodeProcess nodeProcess)
         {
             _nodeProcess = nodeProcess ?? throw new ArgumentNullException(nameof(nodeProcess));
-            if (Environment.GetEnvironmentVariable("JSII_DEBUG") != null)
```

Comment:
What functionality do we lose here?

Reply:
Previously, `STDERR` from anywhere in `node` (i.e: `console.error` from user's apps) were only surfaced to the console of non-JS applications when `JSII_DEBUG` was set to `true`. This means the behavior of a program was *different* if you ran it "pure JavaScript" versus through a jsii binding. Hence I think this was actually a bug and not a feature.